lib/awspec/helper/finder.rb

```diff
@@ -173,6 +173,13 @@ module Finder
       http_wire_trace: ENV['http_wire_trace'] || false
     }
 
+    # We have to set this conditionally after defining `CLIENT_OPTIONS`,
+    # because setting the `endpoint` argument to `nil` results in an error from
+    # the aws sdk.
+    if ENV.key?('aws_custom_endpoint')
```

Comment:
```suggestion
    if ENV.key?('endpoint')
```

I thought `aws_endpoint` or `endpoint` would be better for simplicity

Reply:
Oh, of course, this whole project is about aws! That makes sense to me.